WordPress/Classes/ViewRelated/Domains/Transfer Domains/TransferDomainsWebViewController.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+final class TransferDomainsWebViewController: WebKitViewController {
+
+    private enum Constants {
+        static let url = URL(string: "https://wordpress.com/transfer-google-domains/")!
```

Comment:
We should be using this URL as it's not google domains specific 👍 
```suggestion
        static let url = URL(string: "https://wordpress.com/setup/domain-transfer/intro")!
```